src/views/steps/subviews/service.coffee

```diff
@@ -55,7 +55,7 @@ module.exports = class ServiceView extends ItemView
 
 
         cozy.client.intents
-            .create 'CREATE', 'io.cozy.accounts', slug: @slug
+            .create 'CREATE', 'io.cozy.accounts', { slug: @slug, withCross: false }
```

Comment:
We're still using `withCross`? 😨

Reply:
Yes, this issue is still pending https://github.com/cozy/cozy-ui/issues/198

Reply:
That's not the same `withCross`, this one is for the service, not for the modal :) and it's temporary

Reply:
https://github.com/cozy/cozy-ui/pull/237 ;)